packages/manager/modules/server-sidebar/src/order.constants.js

```diff
@@ -252,6 +252,46 @@ export const ORDER_URLS = {
       PL: 'https://www.ovh.com/manager/sunrise/index.html#/csp2',
       PT: 'https://www.ovh.com/manager/sunrise/index.html#/csp2',
     },
+    dbaas_logs: {
+      CA:
```

Comment:
Order constants are partitioned by IS : https://github.com/ovh/manager/blob/fb660dd670c274697e868830ceb8d4085f06e481/packages/manager/modules/server-sidebar/src/order.constants.js#L4 and https://github.com/ovh/manager/blob/fb660dd670c274697e868830ceb8d4085f06e481/packages/manager/modules/server-sidebar/src/order.constants.js#L296

So `CA,QC, WE,WS` (I notice no ASIA, AU and SG) should be declared in the CA part instead maybe

Reply:
Fixed, thanks.